how2meet/routers/events.py

```diff
@@ -23,7 +23,7 @@ def create_event(event: schemas.EventCreate, db: Session = Depends(get_db)) -> m
 
     Args:
         event: The event to be created.
-        db: The database session. Defaults to Depends(get_db).
+        db: The database session. Defaults to Depends(geft_db).
```

Comment:
I had assumed you were adding this whole sentence or something... but this was just a straight up stray "f" 😂